benchmark/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+
+services:
+  prometheus:
+    network_mode: "host" # only needed if run locally
+    image: prom/prometheus:latest
+    ports:
```

Comment:
host networking and port bindings are incompatible, have you tested if this stack starts?

Reply:
it worked, but I replaced it with `extra_hosts` as suggested off-line